src/ui/component/tag/view.jsx

```diff
@@ -15,7 +15,7 @@ type Props = {
 export default function Tag(props: Props) {
   const { name, onClick, type = 'link', disabled = false } = props;
   const isMature = MATURE_TAGS.includes(name);
-  const clickProps = onClick ? { onClick } : { navigate: `/$/tags?t=${name}` };
+  const clickProps = onClick ? { onClick } : { navigate: encodeURIComponent(`/$/tags?t=${name}`) };
```

Comment:
This should only wrap the `name` variable so the routing still works. Can you change this to 
```
{ navigate: `/$/tags?t=${encodeURIComponent(name)}` };
```

Reply:
Oof sorry about that, I've fixed it